src/lucky_web/page_helpers/number_to_currency.cr

```diff
@@ -0,0 +1,33 @@
+module LuckyWeb::NumberToCurrency
+  DEFAULT_PRECISION       = 2
+  DEFAULT_UNIT            = "$"
+  DEFAULT_SEPARATOR       = "."
+  DEFAULT_DELIMITER       = ","
+  DEFAULT_DELIMITER_REGEX = /(\d)(?=(\d\d\d)+(?!\d))/
+  DEFAULT_FORMAT          = "%u%n"
+
+  def number_to_currency(value : Nil, **options)
+  end
+
+  def number_to_currency(value : Float | Int32, precision : Int32 = DEFAULT_PRECISION, unit : String = DEFAULT_UNIT, separator : String = DEFAULT_SEPARATOR, delimiter : String = DEFAULT_DELIMITER, delimiter_pattern : Regex = DEFAULT_DELIMITER_REGEX, format : String = DEFAULT_FORMAT, negative_format : String = DEFAULT_FORMAT)
```

Comment:
Is there a common style for such long argument lists?

Reply:
Here's what I *think* you can do. Put the `value` argument on a new line and maybe precision too. Then run `crystal tool format` I think that it will put the rest of them on newlines and format them. LMK if that does the trick!